### ISSUE TITLE
When using example DocPlugin(), to_dict() paths contains an instance of Path.

### ISSUE THREAD

**Issue by justai-net** [marshmallow-code/apispec#300]
```
'paths': OrderedDict([(<apispec.core.Path object at 0x10665cda0>,
OrderedDict([('get',
                                      {'responses': {200: {'description': 'A '
                                                                          'list '
                                                                          'of '
                                                                          'all '
                                                                          'appcodes',
                                                           'schema': {'$ref': '#/definitions/appcode'}}},
                                       'summary': 'Gets all appcode',
                                       'tags': ['appcode']})]))]),
```
Wanted
```
'paths': OrderedDict([('/appcode',
OrderedDict([('get',
                                      {'responses': {200: {'description': 'A '
                                                                          'list '
                                                                          'of '
                                                                          'all '
                                                                          'appcodes',
                                                           'schema': {'$ref': '#/definitions/appcode'}}},
                                       'summary': 'Gets all appcode',
                                       'tags': ['appcode']})]))]),
```

Managed to find a fix,
#core.py
```
    def update(self, path):
        if path.path:
            self.path = path.path
        self.operations.update(path.operations)
```

#core.py
```
    def update(self, path):
        if path.path:
            self.path = path.path.path
        self.operations.update(path.operations)
```

**Comment by lafrech** [comment]
Thanks for reporting.

I believe this is a duplicate of https://github.com/marshmallow-code/apispec/issues/234 which is fixed in apispec 1.x beta.

The doc is wrong, because in 1.x beta `Path` was removed. Fix in #301.

**Comment by justai-net** [comment]
Thanks,
